src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTestBase.cs

```diff
@@ -4,12 +4,9 @@
 
 using System.Diagnostics;
 using System.IO;
-using System.Reflection;
 using System.Net.Test.Common;
-
+using System.Reflection;
```

Comment:
Do we still need to use reflection?  We used it when `HttpClient.DefaultRequestVersion` wasn't an approved public API.

But it is now.  So, we can probably remove the use of reflection in this file.

Reply:
Is it available everywhere the tests run? E.g. UAP? (Netfx tests are dead, right?)